life expectancy.Rmd

```diff
@@ -8,50 +8,51 @@ knitr::opts_chunk$set(echo = TRUE)
 ```
 
 
-The data I am working on is about the life expectancy from all contries from year 1800 to 2100, 
+
+### The data I worked on is of the life expectancy from all contries from year 1800 to 2100, from this website:
 [Behavioral Risk Factor Surveillance System (BRFSS)](https://www.cdc.gov/brfss/about/index.htm)
 
 ```{r message = FALSE,warning = FALSE}
 library(tidyverse)
 library(ggplot2)
 ```
 
-Import and take a look at the data
+#### Import and take a look at the data
 
 ```{r}
 life <- read.csv("life_expectancy_years.csv", header = TRUE, stringsAsFactors = FALSE, check.names = FALSE)
 head(life)
 ```
 
-Modify the dataset structure
+#### Modify the dataset structure
 ```{r}
 rownames(life) <- life[,1]
 life$country <- NULL
 ```
 
-Then, I would like to look at the life expactancy trend for China from year 1800 to 2100
+#### Then, I would like to look at the life expactancy trend for China from year 1800 to 2100
 
 ```{r}
 year <- colnames(life)
 year <- as.numeric(year)
 China <- as.numeric(life[c("China"),])
 ```
 
-Then, I would like to look at the life expactancy trend for United States from year 1800 to 2100
+#### Then, I would like to look at the life expactancy trend for United States from year 1800 to 2100
 
 ```{r}
 year <- colnames(life)
 year <- as.numeric(year)
 USA <- as.numeric(life[c("United States"),])
 ```
 
-Combine the subsets into a new table, and esier to draw plots from
+#### Combine the subsets into a new table, and esier to draw plots from
 ```{r}
 new_life <- data.frame(year, China, USA)
 head(new_life)
 ```
 
-Draw the plot
+#### Draw the plot
 ```{r}
 colors <- c("China" = "red", "USA" = "blue")
```

Comment:
This is such a good idea! I did not think to do this but it would be much easier than trying to remember which color you assigned what when doing the code, especially if you are working with a lot of treatment groups (or countries in this case)